apps/judicial-system/web/src/routes/Prosecutor/RestrictionRequest/StepThree/StepThree.tsx

```diff
@@ -18,11 +26,38 @@ export const StepThree: React.FC = () => {
     caseNotFound,
   } = useContext(FormContext)
   const { user } = useContext(UserContext)
+  const { autofill } = useCase()
+  const { formatMessage } = useIntl()
 
   useEffect(() => {
     document.title = 'Dómkröfur og lagagrundvöllur - Réttarvörslugátt'
   }, [])
 
+  useEffect(() => {
+    if (
+      workingCase.requestedCustodyRestrictions &&
+      workingCase.requestedCustodyRestrictions.indexOf(
+        CaseCustodyRestrictions.ALTERNATIVE_TRAVEL_BAN_REQUIRE_NOTIFICATION,
+      ) > -1 &&
+      workingCase.defendants
+    ) {
+      autofill(
+        'requestedOtherRestrictions',
+        formatMessage(
+          rcDemands.sections.custodyRestrictions
+            .requestedOtherRestrictionsAutofill,
+          {
+            genderedDefendant: formatMessage(core.accused, {
```

Comment:
How do you feel about (migrating/use going forward) these kinds of translate messages to use select-format https://formatjs.io/docs/core-concepts/icu-syntax#select-format

Reply:
I love it! 😍 I didn't know about this. I think we should use this and other formatting formats wherever possible!